### PR TITLE
ft(toast-notification):Users should see a toast for new notifications

### PR DESCRIPTION
**Pull request by kalisaNkevin**
#### Questions:

### CODE REVIEW

**Comment by ericus123** [comment]
@kalisaNkevin  rebase

File: src/components/Notifications/toastAlert.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <Alert
+      variant="filled"
+      severity="error"
+      action={
+        <IconButton
+          aria-label="close"
+          color="inherit"
+          size="small"
+          onClick={() => {
+            dispatch(alertActions.error({ message: '' }))
```
Comment (ericus123) [inline src/components/Notifications/toastAlert.js:49]: where does the message come from ???

File: src/components/Notifications/toastAlert.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    <Alert
+      variant="filled"
+      severity="info"
+      action={
+        <IconButton
+          aria-label="close"
+          color="inherit"
+          size="small"
+          onClick={() => {
+            dispatch(alertActions.info({ message: '' }))
```
Comment (ericus123) [inline src/components/Notifications/toastAlert.js:74]: where does the message come from ???
Its empty string ??

File: src/components/Notifications/toastAlert.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    <Alert
+      variant="filled"
+      severity="success"
+      action={
+        <IconButton
+          aria-label="close"
+          color="inherit"
+          size="small"
+          onClick={() => {
+            dispatch(alertActions.success({ message: '' }))
```
Comment (ericus123) [inline src/components/Notifications/toastAlert.js:100]: where does the message come from ???
Its empty string ??

File: src/components/Notifications/toastAlert.js (hunk 2 above)
Comment (ericus123) [inline src/components/Notifications/toastAlert.js:74]: where does the message come from ???
Its empty string ??

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: src/components/Notifications/toastAlert.js (hunk 2 above)
Comment (kalisaNkevin) [reply src/components/Notifications/toastAlert.js:74]: An empty message . What i did here i made the component

File: src/components/Notifications/toastAlert.js (hunk 3 above)
Comment (kalisaNkevin) [reply src/components/Notifications/toastAlert.js:100]: An empty message . What i did here i made the component

File: src/components/Notifications/toastAlert.js (hunk 1 above)
Comment (kalisaNkevin) [reply src/components/Notifications/toastAlert.js:49]: An empty message . What i did here i made the component

File: src/components/Notifications/toastAlert.js (hunk 2 above)
Comment (kalisaNkevin) [reply src/components/Notifications/toastAlert.js:74]: An empty message . What i did here i made the component

**Comment by ericus123** [comment]
@kalisaNkevin this PR doesn't have vercel preview link ???
How can it be tested?

**Comment by ericus123** [comment]
@kalisaNkevin this PR doesn't have vercel preview link ???
How can it be tested?
The build is failing.

**Pull request closed without merging**